tests/test_generate_values_for_insertion_from_df.py

```diff
@@ -1,217 +0,0 @@
-import pandas as pd
-import pytest
-
-from src.utils import generate_values_for_insertion_from_df
-
-
-@pytest.mark.parametrize("df, enrichment, columns, expected_output", [
-    (
-        pd.DataFrame({
-            "FINANCIAL TYPE": ["Type1"],
-            "SYMBOL": ["SYM1"],
-            "SECURITY NAME": ["Security1"],
-            "SEDOL": ["SEDOL1"],
-            "ISIN": ["ISIN1"],
-            "PRICE": [100.0],
-            "QUANTITY": [10],
-            "REALISED P/L": [5.0],
-            "MARKET VALUE": [1000.0]
-        }),
-        {"enrichment_col": "enrichment_value"},
-        [
-            "FINANCIAL TYPE",
-            "SYMBOL",
-            "SECURITY NAME",
-            "SEDOL",
-            "ISIN",
-            "PRICE",
-            "QUANTITY",
-            "REALISED P/L",
-            "MARKET VALUE"
-        ],
-        "('Type1', 'SYM1', 'Security1', 'SEDOL1', 'ISIN1', 100.0, 10, 5.0, 1000.0, 'enrichment_value')"
-    ),
-])
-def test_all_columns_present(df, enrichment, columns, expected_output):
-    result = generate_values_for_insertion_from_df(df, enrichment, columns)
-    assert result == expected_output
```

Comment:
Could use sets to check expected columns exist without depending on the ordering...